wallet/errors.go

```diff
@@ -0,0 +1 @@
+package wallet
```

Comment:
Is this file supposed to be here?

Reply:
Think it got added after running go fmt ./...

Reply:
A file go added by go fmt ?